src/Features/Core/Shared/Extensions/DiagnosticAnalyzerExtensions.cs

```diff
@@ -54,7 +54,12 @@ public static DiagnosticAnalyzerCategory GetDiagnosticAnalyzerCategory(this Diag
                         category |= DiagnosticAnalyzerCategory.ProjectAnalysis;
```

Comment:
by the away, I noticed driver.GetAnalyzerActionsAsync(analyzer).WaitAndGetResult(driver.CancellationToken); this above. why is it using WaitAndGetResult rather than making this Async method?